### PR TITLE
added unittest for inserting into partitioned table that is read only

### PR DESCRIPTION
**Pull request by adrpar**
This depends on PR https://github.com/crate/elasticsearch/pull/56

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java [hunk 1]
```diff
@@ -2106,6 +2107,31 @@ public void testPartitionedColumnIsNotIn_Raw() throws Exception {
...
+                .prepareGetTemplates(templateName).execute().actionGet();
+        Settings templateSettings = templatesResponse.getIndexTemplates().get(0).getSettings();
+        assertThat(templateSettings.getAsBoolean(IndexMetaData.SETTING_READ_ONLY, false), is(true));
+
+        // in order to let junit clean up the ES cluster properly, restore state of ES cluster
+        // otherwise we get a org.junit.runners.model.MultipleFailureException
+        try {
+            execute("insert into parted (id, name, date) values (?, ?, ?)",
+                    new Object[]{1, "Ford", 13959981214861L});
+            throw new RuntimeException("Test failed");
```
Comment (dobe) [inline sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java]: use expectedException rule here, see examples in other tests

**Comment by dobe** [comment]
changes.txt

**Comment by dobe** [comment]
as discussed, check diferent teardown and use sql for settings assertion

**Comment by adrpar** [comment]
moved integration test into a new class, so that it can have its own tearDown method

**Comment by chaudum** [comment]
ack

**Pull request merged**